disnake/permissions.py

```diff
@@ -105,6 +105,11 @@ class Permissions(BaseFlags):
             Returns a new Permissions instance with all enabled permissions from both x and y.
             (Using ``|=`` will update in place).
 
+            .. versionadded:: 2.6
+        .. describe:: Permissions.y | Permissions.z, Permissions(y=True) | Permissions.z
```

Comment:
should be moved to separate section similar to operations on other types

Reply:
https://github.com/DisnakeDev/disnake/pull/615/commits/b4b86b9f5ac6d22f21d4368c63de0ab256b8f8f5